checkov/terraform/checks/resource/aws/ElasticSearchTLSPolicy.py

```diff
@@ -0,0 +1,25 @@
+from typing import List, Any
+
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class ElasticsearchTLSPolicy(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Verify ElasticSearch domain is using an up to date TLS policy"
+        id = "CKV_AWS_197"
+        supported_resources = ("aws_elasticsearch_domain",)
+        categories = (CheckCategories.ENCRYPTION,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self) -> str:
+        return "domain_endpoint_options/[0]/tls_security_policy"
+
+    def get_expected_values(self) -> List[Any]:
+        return ["Policy-Min-TLS-1-2-2019-07"]
```

Comment:
you don't need to set this, if `get_expected_value` has the same content as `get_expected_values`. it will be set for you by the parent class

Reply:
But as soon as they update the policy list you will?

Reply:
yeah, true, lets leave it 😄